Android/app/src/main/java/io/github/project_travel_mate/destinations/CityFragment.java

```diff
@@ -231,8 +233,8 @@ public void onResponse(Call call, final Response response) {
                                         ar.getJSONObject(i).optString("image"),
                                         ar.getJSONObject(i).getString("city_name"),
                                         ar.getJSONObject(i).getInt("facts_count"),
-                                        getString(R.string.interest_know_more), getString(R.string.interest_weather),
-                                        getString(R.string.interest_fun_facts), getString(R.string.interest_trends)));
+                                        mCityContext.getString(R.string.interest_know_more), mCityContext.getString(R.string.interest_weather),
```

Comment:
You can use mActivity only here. No need to create new variable.

Reply:
I created it because you need to fetch the context from the view of fragment. That is why crash was occuring.

Reply:
Use mActivity. mActivity is assigned at the time of attaching. That's the correct way to get context. getActivity() can cause crash.

Reply:
Have made those changes. Please check.